rpc/src/types/errors.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use std::collections::HashMap;
+
+#[derive(Serialize, Deserialize)]
+pub struct ResponseError {
+  title: String,
+  description: String,
+  r#type: String,
+  properties: HashMap<String, Value>,
+  required: Vec<String>,
+  #[serde(rename(
+    serialize = "additionalProperties",
+    deserialize = "additionalProperties"
+  ))]
```

Comment:
If we have time in the future maybe we can make our own macro that automatically does this for all snake case fields

Reply:
This will be removed in the next commit but I agree that we should have a macro for this when we use it

Reply:
mmmmm macros. Would +1 if this problem comes up again